components/org.wso2.carbon.identity.api.server.userstore/org.wso2.carbon.identity.api.server.userstore.v1/src/main/resources/userstore.yaml

```diff
@@ -693,6 +693,10 @@ components:
           type: string
           example: /t/{tenant-domain}/api/server/v1/userstores/SkRCQy1TRUNPTkRBUlk
           description: Location of the created/updated resource.
+        typeName:
```

Comment:
`type` WDYT?

Reply:
We are using the `typeName` as the key in in `/userstores/{userstore-domain-id}` API.